xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/RenamePageIT.java

```diff
@@ -511,10 +510,10 @@ void renamePageRelativeLinkPageAndTranslation(TestUtils setup, TestReference tes
 
         // Make sure the link was refactored in both the page and its translation
         Page newPage = setup.rest().get(new LocalDocumentReference(newSpace, newName));
-        assertEquals("[[" + parent + ".OtherPage.WebHome]]", newPage.getContent());
+        assertEquals("[[OtherPage]]", newPage.getContent());
 
         newPage = setup.rest().get(new LocalDocumentReference(newSpace, newName, Locale.FRENCH));
-        assertEquals("fr [[" + parent + ".OtherPage.WebHome]]", newPage.getContent());
+        assertEquals("fr [[OtherPage]]", newPage.getContent());
```

Comment:
This test is not good anymore: the `OtherPage` doesn't exist so we just check that the unexisting relative link is not updated. But we don't check anymore that link are refactored in a translation page. We need to improve it.